PaperBlossoms/data/README.md

```diff
@@ -0,0 +1,312 @@
+... to schools
+... to titles
+... to bonds
+... to item patterns
+$ python3 PaperBlossoms/data/scripts/add_enums.py -h
+usage: add_enums.py [-h] [--option [{rings,clans,skills,techniques,qualities,equipment,advantages,books,resistance,currency} [{rings,clans,skills,techniques,qualities,equipment,advantages,books,resistance,currency} ...]]]
+
+Utility to add enums to json schema
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --option [{rings,clans,skills,techniques,qualities,equipment,advantages,books,resistance,currency} [{rings,clans,skills,techniques,qualities,equipment,advantages,books,resistance,currency} ...]]
+                        Which enums you want to write to json schemas (defaults to all with no arguments specified)
+```
+
+### Translations
+
+Translations for the UI and the database are stored separately. In this section, I describe how translations are handled within the database.
+
+How translations are stored/added:
+
+- Database translations are stored as CSV files in the [i18n](i18n/) directory.
+- Translations are simply `string` to `string_tr` mappings.
+- All possible strings which could be translated are designated with `*_tr` fields in the database. Paper Blossoms includes a utility to generate a spreadsheet with a list strings to be translated.
+- Translators fill out the spreadsheet with translations and check it in as a new CSV.
+
+How the application fetches translations:
+
+- At application launch, Paper Blossoms automatically loads the appropriate CSV into the `i18n` table in the application database (based on the same logic ).
+- The `i18n` table is part of the view definition for each entity. Every field in the application database that could be translated has a translation field with the `*_tr` suffix, which is populated in the view definition of the entity with the appropriate `i18n.string_tr` value, or the untranslated value if a translation isn't available.
+- During runtime, the application queries the application database, fetching the `*_tr` values.
+
+## Contributing data
+
+Below, I describe how to contribute base data. Customer user data is user-specific, and is not meant to be released with the application. Translations have a [separate process](https://github.com/dashnine/PaperBlossoms/wiki/Translation), and can be provided for both the UI and the database.
+
+Contribution guidelines:
+
+- Never modify `paperblossoms.db` directly. We will only merge PRs after validating JSON data and regenerating `paperblossoms.db`.
+- Coordinate with @dashnine about changes to the application database schema or the JSON schemas. Open [an issue](https://github.com/dashnine/PaperBlossoms/issues/new/choose) or [draft PR](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests#draft-pull-requests) to make your intention to do so explicit.
+- It is polite to refresh `enum`s in JSON schemas if you add any data which go into `enum`s.
+
```

Comment:
Might want to note again that we will reject commits in which user_descriptions are included.